### ISSUE TITLE
gengpgkey - dockerfile fails at 'RUN apt-get update'

### ISSUE THREAD

**Issue by gageoleighton** [mherrmann/fbs#304]
I'm trying to code sign and release my app for Linux using Virtualbox on a windows host. When I try to generate a gpg key it's returning the error below. There were a number of errors when updating package lists so which repository should have the fbs/gpg-generator image?

 * Your operating system(s)
   * Windows 10 running Ubuntu 24.04 in VirtualBox
 * Your Python version
   * 3.12.3 - default in the iso
 * Your fbs version
   * 1.1.0 pro
 * Your PyInstaller version
   * N/A - Docker issue
 * Your PyQt / PySide version
   * N/A but PySide6.7.2
 * A copy of any error messages
Line 8 of the docker file ->
```
ERROR: failed to solve: process "/bin/sh -c apt-get update" did not complete successfully: exit code: 100
Unable for find image 'fbs/gpg-generator:latest' locally
docker: Error response from daemon: pull access denied for fbs/gpg-generator, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
```

 * A (minimal!) script that reproduces the problem you are experiencing.
   * fbs gengpgkey

Thanks!
Gage

**Comment by mherrmann** [comment]
Sorry for the trouble. This is fixed in fbs and fbs Pro version 1.2.3.

**Comment by gageoleighton** [comment]
Thanks, Michael. I suppose I will purchase hobbyist again since there are other code signing and sentry fixes since 1.1.0.
